src/sqlfluff/rules/L028.py

```diff
@@ -14,6 +23,8 @@ class Rule_L028(Rule_L020):
        structs which trigger false positives. It can be enabled with the
        ``force_enable = True`` flag.
```

Comment:
Since you now support structs in this rule do we still need this note?
And the `force_enable` flag?

Also nit but if keeping it, can we update text to "This rule is disabled by default for BigQuery, Hive and Redshift due to their use of"

Reply:
To what should we change the text?

I'd probably leave force enabled flag for now (issue for a bigger release) its a more serious change that could be breaking and probably outside the thought level I can give this before pushing out 0.11.1

Reply:
Sounds good.

At the moment the text just mentions BigQuery, but it's three dialects this affects so think we should mention the other two: "This rule is disabled by default for BigQuery, Hive and Redshift due to their use of..."

Or alternatively should say "This rule is disabled by default for dialects like BigQuery due to their use of..." to avoid having to keep it in sync in future, but do think listing all three is more friendly to users.